mcap_vendor/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.14)
```

Reply:
Based on discussion in WG, Ubuntu Focal has this version from system. This will support ROS1Noetic, and all non-EOL ROS 2 distros.